mitosheet/src/components/taskpanes/AITransformation/AITransformationTaskpane.tsx

```diff
@@ -242,39 +243,82 @@ const AITransformationTaskpane = (props: AITransformationTaskpaneProps): JSX.Ele
     const currentlySelectedParamsIndex = getCurrentlySelectedParamsIndex(props.previousAITransformParams, params);
 
 
-    const aiFeatureDisabled = aiPrivacyPolicyNotAccepted;
-
-    return (
-        <DefaultTaskpane>
-            <DefaultTaskpaneHeader 
-                header="AI Transformation"
-                setUIState={props.setUIState}           
-            />
-            <DefaultTaskpaneBody>
-                {(aiPrivacyPolicyNotAccepted) && 
-                    <>
-                        <p className="text-body-1">
-                            Mito AI is a beta feature. To improve the feature, we collect data used by the AI feature, including: dataframe names, column headers, and cell values.
+    if (!aiPrivacyPolicyAccepted) {
+        return (
+            <DefaultTaskpane>
+                <DefaultTaskpaneHeader 
+                    header="Mito Python Copilot"
+                    setUIState={props.setUIState}           
+                />
+                <DefaultTaskpaneBody>
+                    <p>
+                        Welcome to the Mito Python Copilot powered by OpenAI. Before getting started, take a second to review our privacy policy. 
+                    </p>
+                    <Spacer px={10}/>
+                    <CollapsibleSection title={'What data does Mito Python Copilot collect?'}>
+                        <p>
+                            Mito Python Copilot uses the prompt and metadata about your dataframe to generate code that works in the context of your analysis. Without this information, the Mito generated code will require additional customization.
```

Comment:
metadata is misleading here, I think it's just "uses information about your data"